client/my-sites/site-settings/form-discussion.jsx

```diff
@@ -62,6 +64,30 @@ class SiteSettingsFormDiscussion extends Component {
 		);
 	}
 
+	commentDisplaySettings() {
+		const { isJetpack, isCommentsModuleActive, jetpackVersionSupportsCalypsoSettingsUI } = this.props;
```

Comment:
There isn't a toggle in the calypso settings, just the jetpack version. @MichaelArestad what do you think here?